package.json

```diff
@@ -90,5 +92,16 @@
     "typescript": "^3.7.5",
     "webpack": "4.42.0",
     "xml2js": "^0.4.23"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx,css,json,md}": [
+      "prettier --write",
```

Comment:
Mne sa nepaci ze mame prettier ale nijako ho nekontrolujeme. Kontrolujeme `eslint`, a aj ten vie formatovat cez `eslint --fix`. Nezjednotime to?

Reply:
Ohladom `it.only`: https://www.npmjs.com/package/eslint-plugin-no-only-tests

Reply:
@danielhusar ak funguje `prettier -l`, mohli by  sme nim nahradit `eslint`? (Neviem, my v praci pouivame eslint vsade, nie prettier.)

Reply:
Ja by som pouzil obe. Prettier na formatting, a eslint na logiku.